plugins/Toolbox/src/Toolbox.py

```diff
@@ -646,8 +662,10 @@ def _onRequestFinished(self, reply: QNetworkReply) -> None:
                                 Logger.log("e", "Could not find the %s model.", response_type)
                                 break
 
-                            self._server_response_data[response_type] = json_data["data"]
-                            self._models[response_type].setMetadata(self._server_response_data[response_type])
+                            # Workaround: Do not add Metadata for "installed_packages" check JUST YET
```

Comment:
Why not? In cases of work-arrounds, it's best to also explain why it's there (eg; I can see what you're doing, but in 3 months from now, no-one will remember the why)

Reply:
My bad, sorry, should have been more verbose.
Because I wanted to skip the creation of the designated model, because at this point CURA-6979 asks just for the check of discrepancies between Cura and Cloud in installed packages.
I'm probably gonna come back to revise it later when there is the need for showing the packages that are avaialable/ not available to be synced.